src/server/shared/SharedDefines.h

```diff
@@ -969,9 +969,10 @@ enum SpellEffects
     TOTAL_SPELL_EFFECTS                             = 165
 };
 
-enum SpellCastResult
+// EnumUtils: DESCRIBE THIS
+enum SpellCastResult : uint8
 {
-    SPELL_FAILED_SUCCESS = 0,
+    SPELL_FAILED_SUCCESS = 0, // SKIP
```

Comment:
any reason to skip this?

Reply:
Nah, probably not. It's just not used anywhere but the client doesn't mind the value, so we can include it again.